Makefile

```diff
@@ -44,8 +44,8 @@ CONTAINER_ARCH_TARGETS=$(addprefix container-,$(ALL_ARCHITECTURES))
 container:
 	# Pull base image explicitly. Keep in sync with Dockerfile, otherwise
 	# GCB builds will start failing.
-	docker pull golang:1.16.8
-	docker buildx build -t $(REGISTRY)/metrics-server-$(ARCH):$(CHECKSUM) --build-arg ARCH=$(ARCH) --build-arg GIT_TAG=$(GIT_TAG) --build-arg GIT_COMMIT=$(GIT_COMMIT) .
+	docker pull golang:1.17.1
```

Comment:
I am fine with bumping the golang version, but we should stick to the same minor when backporting since there might be some breaking changes between both minors.

Reply:
Good catch. Damien. I made a cherry-pick from master branch did't notice the change of golang version.

You are right. We should stick to the original minor version.